backend/app/testbot/llm.py

```
@@ -0,0 +1,32 @@
+# llm.py
+import json
+import requests
+from config import GEMINI_API_URL
+
+def ask_llm(prompt):
```

Ruff (0.8.2)

Missing return type annotation for public function `ask_llm`

(ANN201)